deploy/cdk/lambda_stack.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, app, construct_id, **kwargs) -> None:
             f"{construct_id}-data-transfer-fn",
             "../lambdas/data-transfer",
             env={
-                "BUCKET": config.MCP_BUCKETS.get(config.ENV, ""),
+                "BUCKET": config.MCP_BUCKETS.get(config.ENV, config.MCP_BUCKETS.get("stage")),
```

Comment:
The default of "" isn't ideal but I'm wondering if it wouldn't be sensible to use it for the fallback value in case the key 'stage' isn't set. i.e.:
```"BUCKET": config.MCP_BUCKETS.get(config.ENV, config.MCP_BUCKETS.get("stage", ""))```

Reply:
the "stage" key is always set: https://github.com/NASA-IMPACT/veda-data-pipelines/blob/support_migration/deploy/config.py#L15-L18

Reply:
```suggestion
                "BUCKET": config.MCP_BUCKETS.get(config.ENV, config.MCP_BUCKETS["stage"]),
```
If we always expect `stage` to be set, I recommend doing a direct lookup (wherein a `KeyError` will be thrown if somehow it's not set)